configs/test/bigquery/datasets.yaml

```diff
@@ -230,7 +230,7 @@ resources:
     properties:
       datasetId: $(ref.main-dataset.datasetReference.datasetId)
       tableReference:
-        tableId: cross-pollination-statistics
+        tableId: cross_pollination_statistics
       timePartitioning:
         type: DAY
       schema:
```

Comment:
Could you also indent the children under fields: below here for consistency with the other tables.

Reply:
Didn't notice, thanks for pointing that out. Done